phpcs.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
```

Comment:
The same way we have in Zend/Apigility, for example:
https://github.com/zendframework/zend-inputfilter/blob/master/phpcs.xml
https://github.com/zfcampus/zf-rest/blob/master/phpcs.xml
This is because we define more rules here, not only PSR-2, but also forcing to use short array syntax, white spaces rules... etc.
Also in this file we have defined paths to check and excluded files (could be different for each project).